portal/static/js/admin.js

```diff
@@ -891,7 +891,7 @@
                         });
                         documents.forEach(function (item) {
                             var c = item["contributor"];
-                            if (!existingItems[c] && hasValue(c)) { //only draw the most recent, same report won't be displayed
+                            if (!existingItems[c] && c) { //only draw the most recent, same report won't be displayed
```

Comment:
I think you would want to check `c` first?...
```suggestion
                            if (c && !existingItems[c]) { //only draw the most recent, same report won't be displayed
```